tests/tests/legacy_kustomizations/knative-eventing-install/test_data/expected/~g_v1_service_broker-filter.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/component: knative-eventing-install
+    app.kubernetes.io/instance: knative-eventing-install
+    app.kubernetes.io/managed-by: kfctl
+    app.kubernetes.io/name: knative-eventing-install
+    app.kubernetes.io/part-of: kubeflow
+    app.kubernetes.io/version: v0.11.0
+    eventing.knative.dev/brokerRole: filter
+    eventing.knative.dev/release: v0.14.2
+    kustomize.component: knative
+    serving.knative.dev/release: v0.11.0
```

Comment:
v0.14.3?

Reply:
Thanks for catching this, as I didn't even notice the label issues with the tests. Seems the knative kustomization.yaml files in `tests/legacy_kustomizations/` need manual updating, so I went ahead and did that to fix all the generated test labels. This should also fix the knative-eventing  namespace issue in the tests.